init-minio-bucket.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-mc alias set myminio http://minio:9000 minioadmin minioadmin123
+mc alias set myminio http://localhost:9000 minioadmin minioadmin123
```

Comment:
this might be necessary to support ui on local machine, but it's hard to say because i've yet to get test data working

Reply:
i changed this back to `minio:`; the ui api should handle this networking, so it should be able to use the docker network just fine